x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/management/trial_license_complete_tier/upgrade_review_prebuilt_rules.ts

```diff
@@ -0,0 +1,679 @@
+  reviewPrebuiltRulesToUpgrade,
+  patchRule,
+  createHistoricalPrebuiltRuleAssetSavedObjects,
+} from '../../../../utils';
+import { deleteAllRules } from '../../../../../../../common/utils/security_solution';
+
+export default ({ getService }: FtrProviderContext): void => {
+  const es = getService('es');
+  const supertest = getService('supertest');
+  const log = getService('log');
+
+  describe('@ess @serverless @skipInServerlessMKI review prebuilt rules updates from package with mock rule assets', () => {
+    beforeEach(async () => {
+      await deleteAllRules(supertest, log);
+      await deleteAllTimelines(es, log);
+      await deleteAllPrebuiltRuleAssets(es, log);
+    });
+
+    describe(`single line string fields`, () => {
+      const getRuleAssetSavedObjects = () => [
+        createRuleAssetSavedObject({ rule_id: 'rule-1', version: 1, name: 'A' }),
+      ];
+
+      describe("when rule field doesn't have an update and has no custom value", () => {
+        it('should not show in the upgrade/_review API response', async () => {
+          // Install all prebuilt detection rules
+          await createHistoricalPrebuiltRuleAssetSavedObjects(es, getRuleAssetSavedObjects());
+          await installPrebuiltRules(es, supertest);
+
+          // Call the upgrade review prebuilt rules endpoint and check that no rules are eligible for update
+          const reviewResponse = await reviewPrebuiltRulesToUpgrade(supertest);
+          expect(reviewResponse.rules).toEqual([]);
+          expect(reviewResponse.stats.num_rules_to_upgrade_total).toBe(0);
+        });
+      });
+
+      describe("when rule field doesn't have an update but has a custom value", () => {
+        it('should not show in the upgrade/_review API response', async () => {
+          // Install all prebuilt detection rules
+          await createHistoricalPrebuiltRuleAssetSavedObjects(es, getRuleAssetSavedObjects());
+          await installPrebuiltRules(es, supertest);
+
+          // Customize a single line string field on the installed rule
+          await patchRule(supertest, log, {
+            rule_id: 'rule-1',
+            name: 'B',
+          });
+
+          // Increment the version of the installed rule, do NOT update the related single line string field, and create the new rule assets
+          const updatedRuleAssetSavedObjects = [
+            createRuleAssetSavedObject({
+              rule_id: 'rule-1',
+              name: 'A',
+              version: 2,
+            }),
+          ];
+          await createHistoricalPrebuiltRuleAssetSavedObjects(es, updatedRuleAssetSavedObjects);
+
+          // Call the upgrade review prebuilt rules endpoint and check that single line string diff field is returned but field does not have an update
+          const reviewResponse = await reviewPrebuiltRulesToUpgrade(supertest);
+          expect(reviewResponse.rules[0].diff.fields).toMatchObject({
+            name: {
+              base_version: 'A',
```

Comment:
nit: What do you think about grouping `base_version`, `current_version`, `target_version` and `merged_version` together (instead of alphabetical sorting) so that it's a bit more readable at a glance?